pkg/provider/loadbalancer.go

```diff
@@ -233,8 +236,16 @@ func (lb *loadbalancer) createLoadBalancerServicePorts(service *corev1.Service)
 
 func (lb *loadbalancer) getLoadBalancerCreatePollInterval() time.Duration {
 	if lb.config.CreationPollInterval > 0 {
-		return time.Duration(lb.config.CreationPollInterval)
+		return time.Duration(lb.config.CreationPollInterval) * time.Second
 	}
 	klog.Infof("Creation poll interval '%d' must be > 0. Setting to '%d'", lb.config.CreationPollInterval, defaultLoadBalancerCreatePollInterval)
 	return defaultLoadBalancerCreatePollInterval
 }
+
+func (lb *loadbalancer) getLoadBalancerCreatePollTimeout() time.Duration {
+	if lb.config.CreationPollTimeout > 0 {
+		return time.Duration(lb.config.CreationPollTimeout) * time.Second
+	}
+	klog.Infof("Creation poll interval '%d' must be > 0. Setting to '%d'", lb.config.CreationPollTimeout, defaultLoadBalancerCreatePollTimeout)
```

Comment:
This seems like the type of error message we'd only want to see if a user configured something incorrectly, and not if  the user chose to use default values.

would it make sense to make the lb.config.CreationPollTimeout an int pointer, so you could tell if the field was set or not, and only display a warning if the field was set but not set to `> 0` value?

Another option would be to modify the message to only say `using default value '%d' for creation poll timeout`, and not indicate anything that looks like a user configuration error in the log message.

Reply:
I will do the same for Interval, at the end the fact that is a pointer do not break the current configs at yaml level (I think no one is consuming this programatically)